### DISCUSSION TITLE
is the force feedback working in this project?

### DISCUSSION THREAD

**Discussion by pickybl2020** [averycblack/mac-precision-touchpad#4] [General]
i was led here from the imbushuo channel in the hopes that you might have get the force feedback working(?)
so I'm just checking before proceeding into removing the old and getting with a new one

also are there any plans for an automated installer that will check updates in the future?

thanks in advance

**Comment by averycblack** [comment]
I'm not entirely sure what you mean by force feedback. Do you mean being able to vary the feedback when clicking?
I did try to do something similar, but was having issues telling the trackpad when to use the haptics. I've got a non-working prototype in #2 

I have no plans on an installer.
